tests/qa/nightly.json

```diff
@@ -0,0 +1,22 @@
+{
+  "service_id": "single_module_test_cycle",
+  "name": "redisai automation-testing",
+  "properties": {
+    "sut_version": "master",
+    "email_recipients": "redisaidev-aaaacob2o7eeecrhkqwua77kku@redislabs.slack.com",
```

Comment:
this is our slack channel?

Reply:
It is. You can generate an email address from Slack: https://slack.com/intl/en-il/help/articles/206819278-Send-emails-to-Slack#:~:text=Click%20the%20details%20icon%20in,email%20address%20for%20the%20conversation.